crates/sargon/src/system/sargon_os/transactions/sargon_os_transaction_analysis.rs

```diff
@@ -582,7 +582,11 @@ mod transaction_preview_analysis_tests {
             )
             .await;
 
-        assert_eq!(result, Err(CommonError::UnknownAccount))
+        // Just asserts that the execution path reached GW preview call
+        assert!(matches!(
+            result,
+            Err(CommonError::ExecutionSummaryFail { .. })
+        ))
```

Comment:
This test is still located among the "tests that fail for some reason" group. I would move it under `success()` one, and probably add the needed stubs so that we don't check for an error (which may lead to think this should fail).